missioncontrol/openapi/openapi.yaml

```diff
@@ -1240,17 +1483,15 @@ components:
           description: the exit code of the process
           type: integer
         stdout:
-          description: a link to the stdout log
+          description: a cid of the stdout log file
```

Comment:
This would be a great spot to return a URI.

Reply:
(in case we outsource the file location in the future)

Reply:
I flip/flopped on this a bit. Decided against mainly because we don't have a url generator function AFAIK (nothing else on the API returns URLs yet)

Reply:
I lean toward URI here because the link will be of interest to clients.
If we don't put a URI here, we'll likely need to construct it in the UI. The user is going to want access to a file link in order to view the contents directly in their browser (to read logs, etc).

Reply:
I don't have a strong preference on this one. I'll mark it as resolved, just wanted to capture my thoughts. I think there are some cool features in openapi 3.0 to support linking / link generation but I haven't had the time to dig in and integrate them in connexion.